docs/internals/contributing/writing-code/javascript.txt

```diff
@@ -0,0 +1,80 @@
+==========
+JavaScript
+==========
+
+While most of Django core is Python, the ``admin`` and ``gis`` contrib apps contain JavaScript code.
+
+Please follow these coding standards when writing JavaScript code for inclusion in Django.
+
+Code style
+----------
+
+- Please conform to the indentation style dictated in the ``.editorconfig`` file.  We recommend using a text editor with `EditorConfig`_ support to avoid indentation and whitespace issues.  Most of the JavaScript files use 4 spaces for indentation, but there are some exceptions
```

Comment:
I think it might be best to mention EditorConfig in both `internals/contributing/writing-code/coding-style.txt` and this new JavaScript-specific coding style file.  It is important for all code style, not just JavaScript or Python.  For example, the HTML files use 2 spaces for indentation.

My reasoning for keeping a separate JavaScript-specific documentation file:

- This file will be useful when the JavaScript tests are added, to explain how they are written and executed
- This file could become more verbose as more JS code style rules are decided upon (e.g. having a space between `)` and `{` as discussed in another comment.